documentation/decisions/adr/2023-12-15-deliverable-reporting-strategy.md

```diff
@@ -0,0 +1,235 @@
+# 30k ft deliverable reporting strategy
+
+- **Status:** Active <!-- REQUIRED -->
+- **Last Modified:** 2023-12-18 <!-- REQUIRED -->
+- **Related Issue:** [#854](https://github.com/HHS/simpler-grants-gov/issues/854) <!-- RECOMMENDED -->
+- **Deciders:** <!-- REQUIRED -->
+  - Lucas Brown
+  - Sarah Knopp
+  - Sumi Thaiveettil
+  - Aaron Couch
+  - Esther Oke
+  - Billy Daly
+- **Tags:** analytics, github
+
+## Context and Problem Statement
+
+In order to report on the progress we're making toward our 30,000 foot deliverables (30k deliverables), we need to be able to connect those deliverables to the ground-level tasks that need to be completed for delivery. Being able to reliably identify which tasks are required for a given deliverable not only enables us to report key metrics like burnup or percent completion for each deliverable, it also allows us to understand which deliverables are being worked on within a given sprint.
+
+Currently, however, we do not have a consistent strategy for indicating which 30k deliverable a given issue is associated with. For custom reporting, we have been using the milestone that an issue is assigned to as a proxy for its 30k deliverable, and for our sprint board, we've been using issue labels to drive filtering and reporting. The fact that we represent this hierarchy differently across reporting channels creates an additional maintenance overhead and increases the likelihood of presenting conflicting information between reports.
+
+Additionally, as the number of 30k deliverables defined for the project grows, we'll need to adjust our strategy for including them in project reporting. By default, we include all 30k deliverables in our reports, but this results in a large chart where in which many deliverables have no points or issues assigned to them. In order to more accurately reflect the status of our deliverables, we need to determine when and how to include them in reporting.
+
+In light of these needs, the goal of this ADR is to:
+
+1. Recommend a consistent strategy for assigning issues to a parent 30k deliverable across reporting channels
+2. Recommend an approach to limiting the number of 30k deliverables that appear in our reports
+
+## Decision Drivers <!-- RECOMMENDED -->
+
+### Assigning issues to deliverables
+
+Our recommended strategy for assigning issues to a given 30k deliverable should consider the following criteria:
+
+- Users assigning an issue to a 30k deliverable should only need to update that value in one place
+- Users should be able to search or filter for the issues assigned to a given 30k deliverable
+- Users should be able to group by 30k deliverables in the sprint board and in [GitHub insights reporting][github-insights]
+- The relationship between issues and 30k deliverables should be consistently represented across all reports
```

Comment:
It might be worth noting something about: we do NOT need to have every issue created before we start reporting on 30k deliverables. I'm guessing there's a fear of seeing "burn up" on the charts (let's say we start working on 30k deliverable A, and it shows it's 20% done, but that's only because we haven't created enough tickets for it, so we'll actually see progress go backwards for a while -- next week it shows 10% done, and the following week it shows 5% done, etc). 

Personally, I think that's okay. It will require some messaging with executive stakeholders, but I think they understand as well. I enjoy seeing the inflection point where we stop burning up and start really burning down towards the finish line. 

But this can definitely be an area for further discussion!

Reply:
@lucasmbrown-usds one option would be to "t-shirt" size the 10Ks that don't have tasks associated with them. We could do that by actually assigning a high point value to the 10K ticket. Then the overall LOE for the 30K would be part of the reporting and the total points and % done wouldn't be so variable. Once a 10K has tasks, the "t-shirt" size could be removed so the points for those aren't counted twice.

Reply:
Good point! I'm also not too worried if we show some burn-up and % done is inaccurate in the early days of working on a deliverable.

Reply:
I left this as is. Did we still want to add further notes here about pointing at various stages throughout the lifecycle of the deliverable?